tests/e2e/specs/out-of-game/api-status.spec.js

```diff
@@ -0,0 +1,32 @@
+const packageVersion = require('../../../../package.json').version;
+
+import { setupGameAsP0 } from '../../support/helpers';
+
+describe.only('API Status Page', () => {
```

Comment:
Love that you did tests for this, awesome! 👏🏼

Reply:
Bump for test love. Let's remove the `.only()` so that if we wanted to add an additional describe block here it could run
```suggestion
describe('API Status Page', () => {
```